python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -877,9 +877,17 @@ def info(self, full=False):
                 s += ", %i other concurrent sessions:\n * %s" % (len(other_sessions), "\n * ".join(other_sessions))
         return s
 
+    def other_session_active(self):
+        """ Returns a True/False for if other sessions have been detected from the Repository """
+        if self.repository.get_other_sessions():
+            return True
+        else:
+            return False
+
     def print_other_sessions(self):
-        other_sessions = self.repository.get_other_sessions()
-        if len(other_sessions) > 0:
+        """ Prints a warning if other active sessions have been detected """
+        if self.other_session_active():
+            other_sessions = self.repository.get_other_sessions()
```

Comment:
I'd rather not add new functions which are doing nothing but `return bool(self.repository.get_other_sessions())`. It's adding complexity to the API when we should we working to simplify wherever possible.

In `Repository_runtime.py` the code could just read:
```python
if registry.repository.get_other_sessions():
    logger.warning("Multiple Ganga sessions detected. The Monitoring Thread is being disabled.")
    logger.warning("Type 'enableMonitoring' to restart")
    setConfigOption('PollThread', 'autostart', False)
```